client/meson.build

```diff
@@ -0,0 +1,21 @@
+deps = [
+	cairo,
+	pango,
+	pangocairo,
+	wlroots,
+	wayland_client,
+]
+
+if gdk_pixbuf.found()
+	deps += [gdk_pixbuf]
```

Comment:
You don't need to check if the dependency is found. Since it's an optional dep, adding it to the list will just work.

>You can pass the opt_dep variable to target construction functions whether the actual dependency was found or not. Meson will ignore non-found dependencies.

http://mesonbuild.com/Dependencies.html

Reply:
woohoo